src/Model/Item.php

```diff
@@ -1160,26 +1158,28 @@ public static function getIdAndNickByGuid($guid, $uid = 0)
 
 		// Does the given user have this item?
 		if ($uid) {
-			$r = q("SELECT `item`.`id`, `user`.`nickname` FROM `item` INNER JOIN `user` ON `user`.`uid` = `item`.`uid`
-				WHERE `item`.`visible` = 1 AND `item`.`deleted` = 0 AND `item`.`moderated` = 0
-					AND `item`.`guid` = '%s' AND `item`.`uid` = %d", dbesc($guid), intval($uid));
-			if (DBM::is_result($r)) {
-				$id = $r[0]["id"];
-				$nick = $r[0]["nickname"];
+			$items = dba::p("SELECT `item`.`id`, `user`.`nickname` FROM `item`
```

Comment:
You can use `dba::fetch_first` for that instead.

Reply:
Currently not, since there is a join in the query.

Reply:
`dba::selectFirtst` can't be used indeed, but `dba::fetch_first` (notice the wrong case) is a nicer way to write `dba::fetch(dba::p($sql, $params))`.

Reply:
You are right.